app/assets/stylesheets/darkswarm/_shop-inputs.scss

```diff
@@ -94,14 +105,21 @@ button.bulk-buy-add.variant-quantity {
   }
 }
 
-span.bulk-buy.variant-quantity {
+[type="number"].variant-quantity {
   border: .1em solid $grey-200;
   height: 2.5rem;
   display: inline-block;
-  min-width: 3em;
-  padding: .5em;
+  width: 2.5rem;
+  padding: 0;
   text-align: center;
   vertical-align: top;
+  appearance: none;
+  -webkit-appearance: none;
+  -moz-appearance: textfield;
```

Comment:
It'd be great if we could assess whether it's better to use the `prefixer` mixin in  https://github.com/openfoodfoundation/openfoodnetwork/blob/master/app/assets/stylesheets/admin/globals/mixins.scss#L7-L18 or get rid of it. In any case, I would rely on tooling to do this.